src/category_theory/limits/shapes/regular_mono.lean

```diff
@@ -4,7 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Scott Morrison, Bhavik Mehta
 -/
 import category_theory.limits.preserves.basic
-import category_theory.limits.shapes.kernels
+import category_theory.limits.shapes.equalizers
 import category_theory.limits.shapes.strong_epi
 import category_theory.limits.shapes.pullbacks
 
```

Comment:
Doesn't the docstring in this file need updating?

Reply:
Good point, thanks!